sem_covid/services/enrich_pipelines/base_enrich_pipeline.py

```diff
@@ -1,6 +1,18 @@
+#!/usr/bin/python3
+
+# base_enrich_pipeline.py
+# Date:  01/07/2021
+# Author: Stratulat Ștefan
+
+"""
+    This module aims to define a basic pipeline for enriching a dataset
```

Comment:
Does this module seem to be specific to PWDB? If so then this has to be specified along with what classifiers may exist.